python/tskit/trees.py

```diff
@@ -10491,6 +10491,31 @@ def ld_matrix(
             mode=mode,
         )
 
+    def sample_nodes_by_ploidy(self, ploidy):
+        """
+        Returns an 2D array of node IDs, where each row has length `ploidy`.
+        This is useful when inidividuals are not defined in the tree sequence
```

Comment:
```suggestion
        This is useful when individuals are not defined in the tree sequence
```

Reply:
I sprinkle these in to prove it's not AI. Fixed.